scrapers/sc/__init__.py

```diff
@@ -52,6 +52,8 @@ class SouthCarolina(State):
             "name": "2019-2020 Regular Session",
             "start_date": "2019-01-08",
             "end_date": "2020-06-25",
+            # TODO: Remove below line after successful past-session scrapes
+            "active": True,
```

Comment:
One alternative to doing the `active: True` juggle here, would be to create a new job in the Task Definitions file for SC, that explicitly tells the scraper to scrape the old session(s) only, on a schedule that is just once a week (or something else very sparse). (at least I think you can run a CLI command that executes a scrape for a particualr session, regardless of whether it is marked active or not)

The advantage to doing it that way (rather than here in this config), are a) downstream users of this code won't accidentally be scraping old sessions and maybe get confused, and b) setting to a sparse schedule will ensure our current session scrape still happens on the normal schedule, but the backfill job just runs once or twice before we shut it off

Reply:
Commit [0bc7f30](https://github.com/openstates/openstates-scrapers/pull/4654/commits/0bc7f30deade9eb71723317f6ef99855b9d16543) makes this an ongoing component of the scraper, and cleans it up into a helper function.

Tasks definitions added in [this PR](https://github.com/openstates/task-definitions/pull/84).